apiary.apib

```diff
@@ -320,6 +317,75 @@ Finds newest accounts.
 
     + Body
 
+### /accounts{?accessLevel}
+
+#### Find accounts [GET]
+
+Finds accounts satisfying provided criteria.
+
+
++ Parameters
+    + accessLevel: "3" (string) - Finds only accounts with access level not smaller than parameter
+
+
++ Request (application/json)
+
+    + Headers
+
+            Accept: application/json
+            Authorization: AMB_TOKEN ...
+
++ Response 200 (application/json)
+
+    + Attributes (object)
+        + results (array) - The latest 100 accounts matching provided criteria
+            + (object) - Account.
+              + registeredBy (string) - The address which has registered the account.
+              + registeredOn (number) -  - UNIX timestamp of account registration.
+              + permissions (array) - A list of permissions granted to this account
+              + accessLevel (number) - Access level of the account. Should be a non-negative integer.
+        + resultCount (number) - Total number of accounts matching provided criteria.
+
+
+    + Body
+
+            {
+                "results": [
+                    {
+                        "address": "0x162a44701727a31f457a53801cd181cd38eb5bbd"
+                        "registeredBy": "0x345b44701723a31f457a534801cd181cd38eb5bcd",
+                        "permissions": ["register_account", "create_entity"],
+                        "accessLevel": 4
+                    }
+                ],
+                "resultCount": 322
+            }
+
++ Response 400 (application/json)
+
+    Validation error
+
+    + Body
+
++ Response 401 (application/json)
+
+    Authentication error
+
+    + Body
+
++ Response 403 (application/json)
+
+    Invalid signature or missing register_account permission
+
+    + Body
+
++ Response 404 (application/json)
```

Comment:
We return 404 on 'get...' endpoints, but on 'find...' endpoint empty array of results is perfectly valid